src/types/Permit2.sol

```diff
@@ -2,7 +2,6 @@
 pragma solidity >=0.8.19;
 
 import { IAllowanceTransfer } from "@uniswap/permit2/interfaces/IAllowanceTransfer.sol";
-import { IPermit2 } from "@uniswap/permit2/interfaces/IPermit2.sol"; // unused but re-exported for convenience
```

Comment:
As the comment says, this is not used but it is re-exported for convenience (i.e. for V2 Periphery users). So we need to add the `solhint-disable` comment in this file